cfgov/ask_cfpb/jinja2/ask-cfpb/answer-search-results.html

```diff
@@ -59,32 +59,35 @@ <h3 class="results-header">{{ _('Showing') }} {{ results.start_index() }}-{{ res
 
         {% elif not results and page.query %}
 
-        <section class="search-results
-                        block
-                        block__flush-top">
-            <div data-gtm_ask-no-results="true">
-                {% if tag %}
-                    <h3 class="results-header">{{ _('No results found for') }} “{{ tag }}”</h3>
-                {% else %}
-                    <h3 class="results-header">{{ _('No results found for') }} “{{ page.query }}”</h3>
-                {% endif %}
+            <section class="search-results
+                            block
+                            block__flush-top">
+                <div data-gtm_ask-no-results="true">
+                    <h3 class="results-header">
+                        {{ _('No results found for') }}
+                        {% if tag %}
+                            “{{ tag }}”
+                        {% else %}
+                            “{{ page.query }}”
+                        {% endif %}
+                    </h3>
```

Comment:
Is this optimization ok @chosak ?

Reply:
Yes, you might be able to optimize even further with

```jinja
{{ _('No results found for') }} “{{ tag or page.query }}”
```

Reply:
Condensed in https://github.com/cfpb/consumerfinance.gov/pull/8100/commits/06b7aa690821df77810419459b2ab85020391c35